### DISCUSSION TITLE
How can I change the response message after signing up with @supabase/auth-ui-react ?

### DISCUSSION THREAD

**Discussion by yardenj2r** [supabase/supabase#12368] [Questions]
Hi there!

I've been able to customize a lot of messages of the "Auth" component imported from @supabase/auth-ui-react by defining the "variables" field in "localization" field in "Auth", but I couldn't find one for the message the user receives after pressing the sign up. How can I change it? The website I'm making is in portuguese, so I need to translate everything.

Thanks!

**Comment by silentworks** [comment]
Currently this is hard coded but I will be working on some auth-ui stuff this week and this is a good one to add to the list as it has been requested before by someone else too.

**Reply by yardenj2r** [reply]
Yeah, I found out this other request a few hours ago. Thanks!!

**Reply by yardenj2r** [reply]
Should I mark this as an answer or wait for the implementation?
